danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/KtxBitBucketServer.kt

```diff
@@ -0,0 +1,5 @@
+package systems.danger.kotlin
```

Comment:
Do we want something for the future? Or we can add it when is needed? 
Looks a little bit a premature optimisation.

Reply:
Up to us! We can delete the empty stuff for now, but i like the idea that we have already a place where to put new extensions if needed